src/preamble.js

```diff
@@ -1019,6 +1023,15 @@ function createWasm(env) {
   addRunDependency('wasm-instantiate');
 #endif
 
+#if LOAD_SOURCE_MAP
+  addRunDependency('source-map');
+
+  function receiveSourceMapJSON(sourceMap) {
+    Module['sourceMap'] = new WASMSourceMap(sourceMap);
```

Comment:
Where possible it's better to not use `Module.*` properties. I think we can define `var wasmSourceMap;` at the top level, and then assign to it from here?